Neuron/Models/Realm/WalletModel.swift

```diff
@@ -24,4 +43,38 @@ class WalletModel: Object {
     override static func primaryKey() -> String? {
         return "address"
     }
+
+    var icon: WalleIconType {
+        get {
+            if let iconName = iconName {
+                return WalleIconType(rawValue: iconName)!
+            }
+            let iconList = WalleIconType.allType
```

Comment:
The logic below this line should have a separate private func and be extracted from `var icon`.